src/_data/jobs.js

```diff
@@ -0,0 +1,53 @@
+const EleventyFetch = require("@11ty/eleventy-fetch");
+
+module.exports = async function () {
+  async function fetchSheetData(sheetId, sheetIndex) {
+    try {
+      const url = `https://api-sheets-bhansa.netlify.app/.netlify/functions/fetchSheetsData?sheetId=${sheetId}&sheetIndex=${sheetIndex}`;
```

Comment:
where's the code of the function being hosted?

Reply:
We can easily host it in our server. I don't want to depend on Netlify for this.

Reply:
Sure, if you're aware and comfortable with other options, do try to host it on your server and switch to it once done. Whatever feels right to you.
Meantime, we can continue to use this.
**Note**: The server only fetches the data from google sheets using a backend service, the code is open and here the repository https://github.com/bhansa/api-sheets-data.

Reply:
In the interest of time, let's continue with the current approach for now.